tests/send-asset-to-itself-keep-balance

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+source ./lib.sh
+
+start_network
+
+itself=$(client config new $(get_node_rpc))
+itself_pubkey=$(echo "$itself" | client config get-public-key)
```

Comment:
Just a random thought - nothing to change: what if the configuration of `client config new` would be a JSON, then you could use `jq` to parse it?

Reply:
Hum, TOML and JSON are equivalent, do you know a tool that translate between the two maybe?
I'm in favor of keeping TOML as it's a "human" format, and provide small cmds to extract needed information (config edit as CLI in short).

Reply:
So after @cgrigis presentation we should not be using these anyway but using environment variables ;) And btw, there is a new kid on the block by Amazon:

https://amzn.github.io/ion-docs/

Just for time travellers: this is chit-chat and has nothing to do with the actual code here. Just thinking out loud...

Reply:
Yep, I'm thinking on how to use env var to do that, such as how to specify the network of nodes for the server (`NODES='$pubkey_1@$address_1 $pubkey_2@$address_2 ...'`?). I'm not sure I agree a 100% with using only env variables, it fits for many cases but maybe not all.

Hoo, `ion` looks interesting, a human readable protobuf maybe?

Reply:
Like all rules, I would suggest to not take them as "absolute" but evaluate when/whether they really make sense for the task at hand ;-)